docs/blueprints/EP035.rst

```diff
@@ -0,0 +1,140 @@
+
+Actions will involve ``flow_stats``, ``of_core`` and Zabbix Wrapper.
+
+Motivation
+==========
+
+Currently ``flow_stats`` is responsible to deal with flow statistics, which are exposed in endpoints:
+
+- ``GET v1/flow/stats`` to get the flows stats by dpid.
+
+- ``GET v1/packet_count/<flow_id>`` for packet count of a specific flow.
+
+- ``GET v1/bytes_count/<flow_id>`` for bytes count of a specific flow.
+
+- ``GET v1/packet_count/per_flow/<dpid>`` for packet count per flow.
+
+- ``GET v1/bytes_count/per_flow/<dpid>`` for bytes count per flow.
+
+However, OF table stats are not supported. 
+
+The goal of this blueprint is to evolve ``flow_stats`` into ``kytos_stats`` for kytos statistics, including OF table stats support.
+
+Specification
+=============
+
+Information about tables is requested with the ``OFPMP_TABLE`` type of the ``OFPT_MULTIPART_REQUEST`` message. 
+The request does not contain any data in the body and the body of the reply consists of an array of struct ``ofp_table_stats``, which contains:
+
+- `table_id`: Identifier of table.
+
+- `active_count`: Number of active entries.
+
+- `lookup_count`: Number of packets looked up in table.
+
+- `matched_count`: Number of packets that hit table. 
+
+Proposed Changes
+----------------
+
+- Create the new napp ``kytos_stats`` from ``flow_stats``. 
+- New event from ``of_core``.
+- Update ``kytos_stats`` to suport OF table stats.
+- Update Zabbix Wrapper.
+- Update the routes in the e2e test suite for `flow_stats` to `kytos_stats`. 
+
+``of_core``
+~~~~~~~~~~~
+
+This napp will support ``MultipartType.OFPMP_TABLE`` and a new event will be published, just so ``kytos_stats`` can subscribe and react accordingly.
+Will be requested by switch and sent only when the switch has confirmed its capabilities:
+
+``controller.switches[dpid].features.capabilities.value & Capabilities.OFPC_TABLE_STATS == Capabilities.OFPC_TABLE_STATS``
+
+Published Events
+----------------
+
+kytos/of_core.table_stats.received
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Event reporting OF table stats
+
+Content:
+
+.. code-block:: python3
+
+   {
+     "switch": <switch>
+     "replies_tables": [<Table stats>]
+   }
+
+
+``kytos_stats``
+~~~~~~~~~~~~~~
+
+- To build `kytos_stats`, one forking from `flow_stats` will be created and then, `flow_stats` will be depreciated. 
+It will allow a period for transitions. The network operator will be able to decide to keep both of them running while related tools are reconfigured (Zabbix).
+
+- An equivalent behavior of flow stats to the current one in ``flow_stats`` will be kept.
+A data structure will be updated with the ``kytos/of_core.table_stats.received`` event. 
+This structure will be indexed by `dpid` and `table_id``, and will allow the following API endpoints to be added:
+
+``GET v1/table/stats`` to get the statistics of tables. Will be provided query parameters filtering by `dpid`` and optionally by `table_id`.
```

Comment:
In rst files, a markdown single backtick is a double backtick, which is a easy to confuse with the markdown notation, if you could use the double backtick on `dpid` and `table_id` just so the format becomes as expected that'd be great. Trivial detail though, but since blueprints will be part of the docs we'd like that their formatting is all set too.

Reply:
Done!